src/test/scala/scala/next/TestOptionOpsExtensions.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.next
```

Comment:
Just to be sure the test is as closed as user code as possible, do you mind using e.g. the package `scala.test`, and then adding an `import scala.next._`?

Reply:
Sure, good idea. Here it is.